onedal/datatypes/tests/test_data.py

```diff
@@ -17,19 +17,82 @@
 import numpy as np
 import pytest
 from numpy.testing import assert_allclose
+from sklearn.datasets import make_blobs
 
-from onedal import _backend
+from onedal import _backend, _is_dpc_backend
+from onedal._device_offload import dpctl_available, dpnp_available
 from onedal.datatypes import from_table, to_table
+
+if dpctl_available:
+    from onedal.datatypes.tests.common import (
+        _assert_sua_iface_fields,
+        _assert_tensor_attr,
+    )
+
 from onedal.primitives import linear_kernel
+from onedal.tests.utils._dataframes_support import (
+    _convert_to_dataframe,
+    get_dataframes_and_queues,
+)
 from onedal.tests.utils._device_selection import get_queues
+from onedal.utils._array_api import _get_sycl_namespace
 
-try:
-    import dpctl
+if dpctl_available:
     import dpctl.tensor as dpt
 
-    dpctl_available = dpctl.__version__ >= "0.14"
-except ImportError:
-    dpctl_available = False
+if dpnp_available:
+    import dpnp
+
+
+data_shapes = [
+    pytest.param((1000, 100), id="(1000, 100)"),
+    pytest.param((2000, 50), id="(2000, 50)"),
+]
```

Comment:
Is 1d data supported? If yes - need to test also.

Reply:
Thank you! Good catch. Done